src/application/app.module.ts

```diff
@@ -1,10 +1,11 @@
 import { Module } from '@nestjs/common';
-import { AppController } from './app.controller';
-import { AppService } from './app.service';
+import { AuthModule } from 'src/auth/auth.module';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { UserModule } from 'src/user/user.module';
 
 @Module({
-  imports: [],
-  controllers: [AppController],
-  providers: [AppService],
+  imports: [AuthModule, UserModule],
+  controllers: [],
+  providers: [PrismaService],
```

Comment:
I think `PrismaService` can be removed
```suggestion
  providers: [],
```